.circleci/config.yml

```diff
@@ -105,7 +105,7 @@ jobs:
   test_80_drupal93_highest:
```

Comment:
Core polishing last bits of compatibility so I filed https://github.com/wodby/php/issues/156

Reply:
Meantime I do maintain PHP in Alpinelinux and there's "drupal-php" image based on it (edge repo, testing)

https://hub.docker.com/r/skilldlabs/php/tags?page=1&ordering=last_updated&name=81 you could use it meantime to test locally